src/python/dxpy/cli/dataset_utilities.py

```diff
@@ -1221,7 +1221,7 @@ def extract_assay_expression(args):
                 )
             )
     elif not args.assay_name:
-        if len(dataset.assays_info_dict["molecular_expression"]) == 0:
+        if dataset.assays_info_dict.get("molecular_expression") is None or len(dataset.assays_info_dict.get("molecular_expression")) == 0:
```

Comment:
Minor enh.:
```
dataset.assays_info_dict.get("molecular_expression") in [None, 0]:
```

Reply:
Would that really work? One is testing existence but the other length == 0

Reply:
@gabriela-reznik Oh, my bad! I've missed the `len` completely. You are right, that would work. Pls disregard this comment.:)